exercise/static/exercise/personalized_points_goal.js

```diff
@@ -0,0 +1,85 @@
+$(document).ready(function() {
+    const $pointsGoalForm = $('#pointsGoalForm');
+    document.getElementById('id_personalized_points_goal_input').focus();
+    $pointsGoalForm.on('submit', function(event) {
+        event.preventDefault();
+        $.ajax({
+            type: 'POST',
+            url: $pointsGoalForm.attr('action'),
+            data: $pointsGoalForm.serialize(),
+            success: function(response) {
```

Comment:
Should we check if the value for the points goal is invalid, i.e., not a digit or a percentage, before sending it to the server? Then there would be less bad requests that have to be processed by the backend.

We would still double-check validity again on the server, and making sure that the value is above `points-to-pass` can be done in the backend only, if it is hard to do it in the frontend.